File: model/src/com/peoples/model/Location.java
package com.peoples.model;


/**
 * CREATE TABLE locations (
	id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
	subject_id INT UNSIGNED NOT NULL,
	created DATETIME NOT NULL,
	longitude DOUBLE NOT NULL,
	lattitude DOUBLE NOT NULL);
	
 * @author Diego
 *
 */
public class Location {
	
	//id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
	private int LOCATION_KEY;
	
	//subject_id INT UNSIGNED NOT NULL,
	private Subject subject;
	
	//created DATETIME NOT NULL,
	private int datetime;
	
	//longitude DOUBLE NOT NULL,
	private double longitude;
	
	//latitude DOUBLE NOT NULL);
	private double latitude;
	

}

File: web-app/cake/app/webroot/js-app/app/model/Location.js
Ext.define('SD.model.Location', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'id',            type: 'int',    useNull: true},
        {name: 'subject_id',    type: 'int',    useNull: true},
        {name: 'longitude',     type: 'float',  useNull: true},
        {name: 'latitude',      type: 'float',  useNull: true},
        {name: 'radius',        type: 'float',  useNull: true},
        {name: 'created',       type: 'date',   dateFormat: Ext.Date.patterns.ISO8601Long},
        'subject'
    ],
    proxy: {
        type: 'rest',
        url : '/rest/locations'
    }
});